a808b51d0d9301fa82390b985c57551966001f9b/Dockerfile.liquid

```diff
@@ -11,12 +11,21 @@ WORKDIR /electrs
 
 RUN cargo build --features liquid --release --bin electrs
 
+FROM debian:stable-slim AS websocat-installer
+
+RUN apt-get update
+RUN apt-get install -y wget
+
+COPY install-websocat.sh /install-websocat.sh
```

Comment:
why not run all inline, instead of requiring an external bash script? more portable

like https://github.com/vulpemventures/docker-elements/blob/master/22.0.2/Dockerfile#L7-L15